src/lib/styles/themes/dark.scss

```diff
@@ -218,4 +218,18 @@
   --negative-emphasis: var(--pink-200);
   --negative-emphasis-contrast: var(--neutral-50);
   --negative-emphasis-light: var(--pink-200-a25);
+
+  --elements-tag-background: var(--night-575);
```

Comment:
I'm wondering if we really need the "elements" prefix here. Do you know what purpose it serves in Figma?

Reply:
Good point; let’s keep it simple and flatter. After syncing with Artem, we decided to remove the elements. Updated.